bin/check-jenkins-health.rb

```diff
@@ -70,7 +70,7 @@ class JenkinsMetricsHealthChecker < Sensu::Plugin::Check::CLI
 
   def run
     https ||= config[:https] ? 'https' : 'http'
-    r = RestClient::Resource.new("#{https}://#{config[:server]}:#{config[:port]}#{config[:uri]}", timeout: config[:timeout]).get
+    r = RestClient::Resource.new("#{https}://#{config[:server]}:#{config[:port]}#{config[:uri]}", timeout: config[:timeout]){|response, request, result| response }.get
```

Comment:
I think it would be better to simply rescue specific scenarios per: https://github.com/rest-client/rest-client#exceptions-see-httpwwww3orgprotocolsrfc2616rfc2616-sec10html